test/e2e/tests/network/update-network.spec.ts

```diff
@@ -360,6 +360,7 @@ describe('Update Network:', function (this: Suite) {
 
         //  Re-open the network menu
         await driver.delay(regularDelayMs);
+        await driver.clickElementSafe({ text: 'Got it', tag: 'h6' });
```

Comment:
What do you think of wait for dialog to close by using: 
```
       await driver.clickElementAndWaitToDisappear({
          text: 'Got it',
          tag: 'button',
        });
```

or `clickElementAndWaitForWindowToClose` ?

Reply:
Thanks @DDDDDanica, I want to keep this `clickElementSafe` cause its not consistently coming and hence this will click on the dialog when it appears else it will not fail the test and proceed.

Reply:
I think @DDDDDanica is raising a good. We've encountered flakiness in the pass precisely because of a pop up not being completely closed, when the next action is performed (See [here](https://docs.google.com/document/d/1oXd5d1X7j14lHLjaRCWjEh3uhndrXQ_46lBuZ9SAu6M/edit#heading=h.q0x5cg6r60mk)).
@hjetpoluru if you mention that sometimes the popup doesn't consistently appears, is it it never appears, or it takes a bit to appear? I think if it never appears we should also open a bug for that.

In the meantime, I think we might want to use a combination of the 2 actions clickElementSafe, as well as wait until the popup completely disappears. For example, you could add the `assertElementNotPresent` for the popup, after clickElementSafe.

What do you think? @hjetpoluru @DDDDDanica

Reply:
After @seaona comment I tried again, and my current observation is that in the latest develop version - dialog does not appear. I need to check further before raising a bug it could be something with the mocks the dialog is appearing. 

Thanks for the input @seaona, after I get more information will be able to implement one you have suggested.